tests/e2e/e2e_common.go

```diff
@@ -329,3 +333,9 @@ func setClusterFlavor(clusterFlavor cnstypes.CnsClusterFlavor) {
 		rwxAccessMode = true
 	}
 }
+
+func setPodImage(varName string) {
+	podimage := os.Getenv(varName)
+	gomega.Expect(podimage).NotTo(gomega.BeEmpty(), "ENV "+podimage+" is not set")
```

Comment:
I'm not sure if `gomega.Expect()` works here, can you do a quick check?

Reply:
GetAndExpectIntEnvVar , GetAndExpectBoolEnvVar all these methods are in the same file. All have gomega.Expect